setup.cfg

```diff
@@ -24,10 +24,10 @@ per-file-ignores =
     openff/toolkit/topology/__init__.py:F401
     openff/toolkit/topology/molecule.py:
     openff/toolkit/topology/topology.py:F821
-    openff/toolkit/typing/chemistry/__init__.py:F401,F403
-    openff/toolkit/typing/engines/smirnoff/__init__.py:F401,F403
+    openff/toolkit/typing/chemistry/__init__.py:F401
```

Comment:
(not blocking) This seems pretty brittle - Are there easy options to have this file not exist?

Reply:
The easiest route would be modifying the code to pass all `flake8` checks - something I'd be happy to do in a separate PR but not something that would be a trivial amount of work. I'd argue most of the checks `flake8` attempts to enforce are good ones.